ui/src/containers/Header/index.tsx

```diff
@@ -82,6 +82,33 @@ const Header: React.FC = observer(() => {
           </TextContent>
         </Grid>
       </Modal>
+
+      <Modal
+        variant={ModalVariant.small}
+        title="Sign In:"
+        isOpen={isSignInModalOpen}
+        onClose={() => setIsSignInModalOpen(false)}
+        actions={[
+          <Button key="cancel" variant="secondary" onClick={() => setIsSignInModalOpen(false)}>
+            Close
+          </Button>
+        ]}
+      >
+        <Grid>
+          <GridItem style={{ marginBottom: '2em' }}>
+            <Button> Sign In with GitHub Enterprise</Button>
+          </GridItem>
+          <GridItem style={{ marginBottom: '2em' }}>
+            <Button> Sign In with GitHub</Button>
+          </GridItem>
```

Comment:
I guess we should keep only `Sign In with Github`

Reply:
You mean , we don't need to keep  `Sign In with GitHub Enterprise` separately

Reply:
yeah... either should be there